_posts/2020-03-13-Azure-Functions-Portal-Preview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Announcing public preview for Azure Functions portal UX"
+tags: 
+    - Function
+    - Preview
+    - UX
+author_name: "Byron Tardif"
+---
+
+## Access the preview
+
+Thew new user experience for **Azure Functions** is now available for all users in the **Azure Portal**.
+
+You can try the new experience by clicking on the "*Preview the new Azure Functions management experience*" banner in your function apps.
+
+![Access the Functions Preview UX]({{ site.baseurl }}/media/2020/03/functions_preview.gif)
+
+## Function App Overview
+
+Once you are in the new experience, you will notice that the new  **Azure Functions** UX is now consistent with the rest of the **Azure Portal**.
+
+This UX is powered by the same [**Azure Functions Runtime API**](https://docs.microsoft.com/azure/azure-functions/functions-runtime-overview) except they are now exposed through [**ARM**](https://azure.microsoft.com/features/resource-manager/). This allows for better caching through the [**Azure Resource Graph**](https://azure.microsoft.com/features/resource-graph/), as well as support for [**RBAC**](https://docs.microsoft.com/azure/role-based-access-control/overview), and enables new scenarios like being able to manage Azure Functions hosted on an [**Internal Load Balancer App Service Environment ASE**](https://docs.microsoft.com/azure/app-service/environment/create-ilb-ase).
+
+The core **Azure Functions** features are grouped in the menu bar under the **Functions (preview)** heading:
+
+![New Function App overview]({{ site.baseurl }}/media/2020/03/functions_app_overview.png)
+
+### Functions
+
+**Functions** lets you list the individual functions within your Function App. From here you can also:
```

Comment:
The double plural sounds awkward to my ears, but might be correct

Reply:
🤷‍♂️ I'll see if @nertim has an opion

Reply:
resolving this based on offline discussion.